### ISSUE TITLE
[BUG]: Required tab stop for aria-activedescendant="" should not fire

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#1539]
### Project

a11y checker engine

### Browser

Chrome

### Operating system

MacOS

### Description

Error triggering:
> The <input> element using 'aria-activedescendant' set to "" is not tabbable

This doesn't really make sense. If the value is empty, it's not really identifying an active descendant.

### Steps to reproduce

https://codesandbox.io/s/modern-rain-vp76mh?file=/index.html

**Comment by shunguoy** [comment]
The 'aria-activedescendant' is not required, so the value doesn't have to be valid

**Comment by shunguoy** [comment]
If the attribute doesn't point to anything (empty or invalid), then ignore the rule.

**Comment by ErickRenteria** [comment]
<img width="1429" alt="Screenshot 2023-09-26 at 3 40 38 PM" src="https://github.com/IBMa/equal-access/assets/62436670/52053ae1-d35b-4f1b-86b6-efa690c46751">
